build/run

```diff
@@ -120,14 +130,19 @@ if [ -z "${DISABLE_NESTED_DOCKER}" ]; then
     -v ${HOME}/.docker/config.json:${BUILDER_HOME}/.docker/config.json \
     -v /var/run/docker.sock:/var/run/docker.sock"
 fi
+DOCKER_API=''
+
+if (( $(bc <<< "$DOCKER_SERVER_API < $DOCKER_CLIENT_API") )); then
+    DOCKER_API="-e DOCKER_API_VERSION=${DOCKER_SERVER_API}"
+fi
```

Comment:
Correct. But I want to make sure that I do not break the other case: older client, newer server. I don't know if the client blows up if we tell it to use a DOCKER_API_VERSION=1.30 while itself only supports 1.24. In that case I'd prefer not to push through a DOCKER_API_VERSION to the client and everything is guaranteed fine.